### ISSUE TITLE
Reviews can be submitted and the reviewers are not aware that they worked

### ISSUE THREAD

**Issue by moorepants** [moorepants/icsc2017.ucdavis.edu#52]
Seems like you have to refresh the browser, or log in / log out to get the recently submitted reviews to show up in the panel.

**Comment by monthubbard** [comment]
Should we put a comment to that effect on the reviewer panel?
Mont

**Comment by monthubbard** [comment]
Should we put a comment to that effect on the reviewer panel? I can do it.
Mont

**Comment by moorepants** [comment]
Yes we should. I just haven't had time to confirm what actually happens.

**Comment by moorepants** [comment]
I've tested this out. I assigned two reviews to my test user and then logged in with that user. The first review goes fine. You click "add review", fill out the form, and click submit. The you are returned to the panel and you see the review in the "my reviews" section. But on the second review, after submitting, the review doesn't show up in the "my reviews" list, so it seems you haven't submitted. Even if you refreshed the page the review still doesn't show up. You have to log out and then log back in for the second submitted review to show up in the list.

**Comment by moorepants** [comment]
I've updated the page with this warning:

Important Note: Due to a bug in the software, if you submit multiple reviews in a row, only the first review will show in the "My Reviews" section until you log out and log back in. When you press submit, the review is added to our database but it will look to you as if it was not successful. We are working on a fix.

and will contact the software author to try to find a fix.

**Comment by moorepants** [comment]
The warning looks like this:
![selection_154](https://cloud.githubusercontent.com/assets/276007/24259618/a5ae8f2a-0fae-11e7-9630-bfe0e1d87a92.jpg)
